cloud/passportauth/src/auth/PassportSetup.ts

```diff
@@ -0,0 +1,73 @@
+import * as express from 'express';
+import * as session from 'express-session';
+import * as passport from 'passport';
+import {Strategy} from 'passport-local';
+import BaseUser from '../user/BaseUser';
+import UserSecService from '../user/UserSec';
+
+export interface Setup {
+  init(app: express.Express): void;
+  setStrategy(): void;
+  setSerializeUser(): void;
+  setDeserializeUser(): void;
+}
+
+class PassportSetup<T extends BaseUser> {
+  protected userSec: UserSecService<T>;
+
+  constructor(protected readonly userSecService: UserSecService<T>) {
+    this.userSec = userSecService;
+  }
+
+  public init(app: express.Express) {
+    app.use(session({secret: 'test', resave: false, saveUninitialized: false}));
```

Comment:
Something along the lines of this could be used for express-session, it can be looked at down the line though, just once we don't forget to add in some basic cookie security.

```
{
      secret: process.env.SESSION_SECRET || 'wfm',
      resave: false,
      saveUninitialized: true,
      cookie: {
        secure: process.env.NODE_ENV !== 'development',
        httpOnly: true,
        path: '/'
      }
    }
```